fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/fallflying/FabricFallFlyingItem.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.item.v1.fallflying;
+
+import java.util.function.Consumer;
+
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.ElytraItem;
+import net.minecraft.item.ItemStack;
+
+/**
+ * An interface that allows implementing items to enable elytra flight for players, when equipped in the chest equipment slot.
+ * While being used, the item will be damaged <b>every 20 ticks</b> through {@link ItemStack#damage(int, LivingEntity, Consumer)}.
+ */
+public interface FabricFallFlyingItem {
```

Comment:
This needs another name, FabricElytraItem seems a lot better to me. The yarn naming is a little odd, that needs fixing there where it can be changed easily.

Reply:
'Fall flying' is both the Yarn name and the NBT name. I don't think it can be changed very easily, unless we want to overrule the NBT name and change both Yarn and Fabric API to use something else like 'gliding'.

Reply:
`FabricElytraItem` and `allowElytraFlight` are also what I'd suggest - sorry for suggesting another name in the first place. :wink:

Reply:
alright ill change the name

Reply:
do we want `allowElytraFlight` or `shouldAllowElytraFlight`

Reply:
went with `shouldAllowElytraFlight`

Reply:
Perfect, this is good for me!